stable/graphite/templates/statefulset.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: apps/v1beta1
```

Comment:
why do you use apps/v1beta1?
apps/v1 is available.

Reply:
Good catch, looks like I was using an old reference! Fixed and pushed.